R/validate.R

```diff
@@ -1,3 +1,7 @@
+verify_condition <- function(condition, env) {
+  if (identical(eval(condition, envir = env), FALSE)) {deparse(condition)}
+  else {NULL}
```

Comment:
I personally like some space around my brackets (e.g., `{ NULL }` not `{NULL}`).